Dockerfile

```
@@ -5,9 +5,10 @@
 FROM rust:latest as cargo-build
 
 COPY . .
-RUN apt-get update
-RUN apt-get install -y cmake
-RUN apt-get install -y clang
+RUN apt-get update && apt-get install -y \
+  cmake \
+  clang \
+  protobuf-compiler
```

I have generally seen docker scripts using apt clean out the lists directory like this
```suggestion
RUN apt-get update && apt-get install -y \
  cmake \
  clang \
  protobuf-compiler \
  && rm -rf /var/lib/apt/lists/*
```